platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Submission.java

```diff
@@ -10,7 +10,9 @@
  */
 @Entity
 @Table(name = "submission")
-public class Submission extends IdentifiableEntity {
+public class Submission extends IdentifiableEntity implements StoredObject{
+    public static final String BUCKET = "coduno";
```

Comment:
remove it from the config and hardcode it into an entity ? i dont think this is a good idea - we should at least define whats in "coduno"